test/Jsr292/src/com/ibm/j9/jsr292/indyn/IndyTest.java

```diff
@@ -211,17 +218,25 @@ public void test_indyn_call_Illegal_crosspackage_method () {
 	//Negative test : invalid bootstrap method
 	@Test(groups = { "level.extended" })
 	public void test_indyn_invalid_bootstrap_method() {
-		
 		boolean bootstrapMethodError = false;
+		boolean noSuchMethodError = false;
 		
 		try {
 			String s = com.ibm.j9.jsr292.indyn.GenIndyn.invalid_bootstrap(new Object());
-		} catch ( BootstrapMethodError e ) {
+		} catch (BootstrapMethodError e) {
```

Comment:
Can the version check be moved into the catch blocks as there is slightly less state to manage then:

i.e.:
```
} catch (BootstrapMethodError e) {
  if (!isJava8()) {
    Assert.fail("[Java 8] BootstrapMethodError not thrown when invalid bootstrap method is used");
  }
  return;
} catch (NoSuchMethodError e) {
  if (isJava8()) {
    Assert.fail("[Java 9 or greater] NoSuchMethodError not thrown when a non-existent bootstrap method is used");
  }
  return;
}
Assert.fail("BootstrapMethodError (Java 8) or NoSuchMethodError (java 9+) should have been thrown");
```

Reply:
done.